components/zcash_address/src/kind/unified.rs

```diff
@@ -241,7 +214,7 @@ impl TryFrom<(&str, &[u8])> for Address {
                     length
```

Comment:
"address" -> "item" on the preceding line.

Reply:
Resolved in #465